lms/views/decorators/h_api.py

```diff
@@ -106,6 +111,41 @@ def wrapper(context, request):
     return wrapper
 
 
+def upsert_group_info(wrapped):
+    """Create or update the :class:`lms.models.GroupInfo` for this launch."""
+
+    @functools.wraps(wrapped)
+    def wrapper(context, request):
+        if not context.provisioning_enabled:
+            return wrapped(context, request)
```

Comment:
`provisioning_enabled` is a feature flag that's scheduled to be removed soon, so we should soon be able to remove all these `provisioning_enabled`-based `if`'s. For now they're still getting peppered around though

Reply:
Why do we want to disable this if provisioning is enabled?

Reply:
We're actually disabling it if provisioning is _not_ enabled. When provisioning is disabled the entire groups integration is disabled (no group even gets created in h)